src/Options.ts

```diff
@@ -35,7 +35,7 @@ let OptionsVals: IOptions = {
     setTimeout: setTimeout.bind(null),
     clearTimeout: clearTimeout.bind(null),
 
-    shouldComponentUpdateComparator: _.isEqual.bind(_),
+    shouldComponentUpdateComparator: () => true,
```

Comment:
Should this be `false` instead? This comparator interprets "true" as "nothing changed", right?